.babelrc

```diff
@@ -4,14 +4,17 @@
       "pragma": "createElement"
     }],
     ["@babel/preset-env", {
+       "bugfixes": true,
```

Comment:
Can you help me understand this property?

Reply:
It is a bit subtle, but essentially it avoids issues where a natively supported feature gets compiled down to ES5 because some specific browser version had a bug in an edge case. The manual explains it better - https://babeljs.io/docs/en/babel-preset-env#bugfixes